### ISSUE TITLE
Columns block:  Content is output twice when More Block is added inside column 

### ISSUE THREAD

**Issue by jerrysarcastic** [WordPress/gutenberg#26839]
**Describe the bug**
When the more block is added within the column block, it causes the content that is added above the more block to be output twice when displayed on any archive page.  

Video screencast of the issue:  https://d.pr/v/uka5mh

Screenshot:

![Screen Shot 2020-11-09 at 10 45 00 AM](https://user-images.githubusercontent.com/1967448/98594370-a683af80-2278-11eb-8f8e-805172c7fb48.png)

**To reproduce**
1. Create test post and add columns block with text
2. Add More block within any column, then publish
3. Visit any archive page and see that content above the more block is output twice

**Expected behavior**
To see the content only once

**Screenshots**

**Editor version (please complete the following information):**
- WordPress version: 5.5.3
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? Issue is present with or without Gutenberg plugin

**Desktop (please complete the following information):**
 - OS: Mac OS 10.15.4
 - Browser Chrome 
 - Version Version 86.0.4240.183 (Official Build) (x86_64)

**Comment by stokesman** [comment]
Likely has the same root cause as this issue #22718

**Comment by talldan** [comment]
I'll close as a duplicate of #22718
